### DISCUSSION TITLE
Hmt-2250-6t + 2 batteries lifepo4 12,8v

### DISCUSSION THREAD

**Discussion by chrisBer31** [hoylabs/OpenDTU-OnBattery#1278] [General]
Hello,

I have setup an opendtu on battery 2024.08.18 with one hmt-2250-6t, one victron mppt 150/45 and smartshunt with 2 batteries eco worthy lifepo4 12v 280ah.

I have an issue with power efficiency on ac output.
All DC+ and DC- cables have the same length to the busbar and connected to each channel on HMT-2250-6t.

Base load is setup to 200w it mean 66w per mppt of HMT.

When all 6 channels are connected to battery ac power output can't produce 200w. It produce around 130w with fluctuations to aquieve 200w. So the DC power on channels 1 and 2 are not converted to produce around 200w in ac power output. The HMT efficiency between DC power and ac power is 60%.

When only 4 channels are connected to battery (Ch 3-4-5-6) the HMT produce 200w in ac power output and the efficiency of the HMT is 95% so all DC power as input is correctly convert.
With only 4 channels are connected to battery (Ch 3-4-5-6) i have setup the base load to 450w the efficiency still 95% so 320w in DC input are convert to 300w to ac power output.

I have tried to switch cables between CH5-6 to CH1-2 the problem still persist.

I also have try to check that CH1-2 connected directly to solar modules with an opendtu without battery no issue the HMT convert DC input power to ac output power with 95% efficiency. I have test also with CH1-3 same results and in second time with CH2-4 same results DC input power is correctly convert.

Anyone have a similar issue with 24v battery and hoymiles microinverter ? There is a solution to solve it ? Or the microinverter have a faillure ?

Thanks for you help

**Comment by Manos1966** [comment]
> Anyone have a similar issue with 24v battery and hoymiles microinverter 

Almost everyone! Not this issue but issues in general when connecting 24V

Solution: move to 48V (yes, I know, it will be painfull, sorry)

Solution: You have a very low consumption (200W) It is 10% of the max power of your HOYMILES. Just use only the four chanels instead of six.

The real issue is actually that HOYMILES was not made to handle well 24V.
Solar Panels have much higher voltage, usually 35V and higher.
The real issue is that HOYMILES does NOT CALCULATE CORRECTLY the absorbed power.
You can identify this by connecting a measurement instrument (Shelly PM) at the AC exit of your HOYMILES.
IF you are lucky, you may find out the HOYMILES is deliverung actually 200W although it reports 130W.

**Reply by spcqike** [reply]
In addition, I guess 200W or 66W per mppt isn’t that much. Idk but I guess the Hmt-2250 behaves like my hms-2000, which can’t produce limits <15%. Well, it can, but it only delivers like 260W when I ask for 300W. It just starts to produce my desired limit, when it’s > 300W or 15%.
